TelepresenceBot/mobile/src/main/AndroidManifest.xml

```diff
@@ -47,22 +47,12 @@
       android:icon="@mipmap/ic_launcher"
       android:label="@string/bot" />
 
-    <receiver android:name="com.novoda.tpbot.bot.UsbReceiver">
```

Comment:
We aren't interested in receiving updates when the app is not in the foreground.